spec/presenters/rejected_application_choice_presenter_spec.rb

```diff
@@ -0,0 +1,167 @@
+require 'rails_helper'
+
+RSpec.describe RejectedApplicationChoice do
```

Comment:
The name of this spec implies we are testing `RejectedApplicationChoicePresenter` but the presenter class name is `RejectedApplicationChoice`. The convention seems to be to use `Presenter` in the class name.